.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Should we put this to `weekly` ? I moved to weekly for `hub` just because with npm it was generating a lot of PR often.. but maybe we are fine here.